infra/aws/terraform/cncf-k8s-infra-aws-capa-ami/oidc.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_iam_openid_connect_provider" "github" {
```

Comment:
FYI - we do have this https://github.com/terraform-aws-modules/terraform-aws-iam/tree/master/modules/iam-github-oidc-provider

Also, this is set ONCE per account - not sure if this would be the right place or if there is somewher where account level settings are created cc @ameukam

Reply:
Great, i wasn't aware of these. Thanks @bryantbiggs . I will update to use the modules instead of the handcrafted versions.

This is account is purely for the CAPA amis, so seems reasonable to have it here.

Reply:
Updated to use this module.

Reply:
This looks technically fine but we need to double check with the Github admin team if we are allowed to do this.
(Kubernetes organizations are not the responsibilities of the Infra SIG)

Reply:
Yes.

Reply:
Feedback from #github-management is that we shouldn't use OIDC from GHA in this way. Prow with IRSA should be used.

I will update the PR accordingly.